x-pack/plugins/event_log/server/es/cluster_client_adapter.test.ts

```diff
@@ -173,6 +173,7 @@ describe('createIndex', () => {
     await clusterClientAdapter.createIndex('foo');
     expect(clusterClient.callAsInternalUser).toHaveBeenCalledWith('indices.create', {
       index: 'foo',
+      body: {},
```

Comment:
I'm curious why we added the extra body here.  I was frankly curious I could create an index WITHOUT a document to begin with, but I'm wondering if this change was deliberate, and if so, why was it done.

Reply:
This is to set additional settings when creating the initial index. I need this to setup the alias when creating the initial index now that it's not done with the index template anymore.

Reference: https://github.com/elastic/kibana/pull/58010/files#diff-88aee8214a0ea2006f946be367b1d715R66